index.html

```diff
@@ -9,9 +9,88 @@
 </head>
 
 <body>
-  Hello world Test
-    <a href = 'game.html'>test</a>
-    <a href = 'boxes.html'>box</a>
+
+    <!-- Boxes -->
+    <div class='rowdiv'>
```

Comment:
Lets put a div around all the boxes and all the keyboard to section the page into 2 main divs for readability and styling purposes down the line. <div class='boxes'> and <div class='keyboard'>